vespa/test_integration_docker.py

```diff
@@ -236,6 +237,129 @@ def test_data_operation(self):
             },
         )
 
+    def test_batch_feed(self):
+        self.vespa_docker = VespaDocker(port=8089, disk_folder=self.disk_folder)
+        app = self.vespa_docker.deploy(application_package=self.app_package)
+
+        #
+        # Create and feed documents
+        #
+        num_docs = 100
+        docs = []
+        for i in range(num_docs):
+            schema = "msmarco"
+            id = f"{i}"
+            title = f"title for document {i}"
+            body = f"body for document {i}"
+            fields = {"id": id, "title": title, "body": body}
+            docs.append((schema, id, fields))
+        app.feed_batch(docs)
+
+        # Verify that all documents are fed
+        result = app.query(query="sddocname:msmarco", query_model=QueryModel())
+        self.assertEqual(result.number_documents_indexed, num_docs)
+
+    def test_data_operations_async(self):
```

Comment:
@lesters I get the following warning when I run the async tests:

```
/Users/tmartins/.local/share/virtualenvs/pyvespa-UOfd_sG0/lib/python3.7/site-packages/urllib3/util/url.py:421: ResourceWarning: unclosed <socket.socket fd=15, family=AddressFamily.AF_UNIX, type=SocketKind.SOCK_STREAM, proto=0, raddr=docker.sock>
  fragment=ensure_type(fragment),
``` 
Could it be that the `self.aiohttp_session.close()` is not closing all the used connections?